fixieai/cli/agent/commands_test.py

```diff
@@ -176,45 +248,211 @@ def test_init_validation_prompting():
         assert config.more_info_url == ""
 
 
+def test_temporary_revision_replacer(mock_graphql):
```

Comment:
Scratching my head about what you're doing here. Mind adding some comments?

Reply:
Ha, fair enough :)